app/components/news/news.hbs

```diff
@@ -0,0 +1,12 @@
+<h1 class="sidebar__title">News</h1>
+<section class="news">
+    <News::TagSwitcher @model={{@model}} @onItemClick={{@onItemClick}} @onResetClick={{@onResetClick}} />
+
+    {{#each @model.news as |model|}}
+        <News::NewsSingle @model={{model}} @onItemClick={{@onItemClick}} />
```

Comment:
Why do you use "model" prop name, I suggest naming it article. Model is too abstract